lor_deckcodes/encode.py

```diff
@@ -21,7 +21,7 @@ def _get_set_faction_combinations(cards: List["CardCodeAndCount"]):
 def _encode_card_block(data_stream: BytesIO, cards: List["CardCodeAndCount"]) -> None:
     set_faction_combinations = _get_set_faction_combinations(cards)
     write_varint(data_stream, len(set_faction_combinations))
-
+    set_faction_combinations = sorted(set_faction_combinations, key=lambda y: (len([card for card in cards if card.faction == y[1] and card.set == y[0]]), y[0], y[1]))
```

Comment:
This should be part of `_get_set_faction_combinations` and I need more context for this